index.d.ts

```diff
@@ -152,6 +153,8 @@ declare class Context {
   static create(defaults: contextDefaults): any;
   retryAttempts: Array<RetryAttempt>;
   addPlugin(plugin: Plugin): Context;
+
+  redirect?: 'follow'|'manual'|'error';
```

Comment:
I think we need to add `redirect` inside the `defaults` type a few lines below this one

Reply:
It should now look something like this:
```ts
export type TransportOptions = {
  agentOpts?: https.AgentOptions,
  defaults?: {
    json?: boolean
    timeout?: number
    compress?: boolean
    proxy?: string
    redirect?: 'follow'|'manual'|'error'
  }
}
```